tests/business/gitlfs_test.py

```diff
@@ -0,0 +1,111 @@
+"""Tests for GitLFS."""
+
+from unittest.mock import ANY, patch
+
+import pytest
+import respx
+from httpx import AsyncClient
+
+from mobu.services.business.gitlfs import GitLFSBusiness
+
+from ..support.gafaelfawr import mock_gafaelfawr
+from ..support.gitlfs import (
+    flock_message,
+    no_git_lfs_data,
+    uninstall_git_lfs,
+    verify_uuid_contents,
+)
+from ..support.util import wait_for_business
+
+
+@pytest.mark.asyncio
+async def test_run(client: AsyncClient, respx_mock: respx.Router) -> None:
+    mock_gafaelfawr(respx_mock)
+
+    # Set up our mocked business
+    with patch.object(
+        GitLFSBusiness,
+        "_install_git_lfs",
+        side_effect=uninstall_git_lfs,
+        autospec=True,
+    ):
+        with patch.object(
+            GitLFSBusiness,
+            "_check_uuid_pointer",
+            side_effect=verify_uuid_contents,
+            autospec=True,
+        ):
+            with patch.object(
+                GitLFSBusiness,
+                "_add_git_lfs_data",
+                side_effect=no_git_lfs_data,
+                autospec=True,
+            ):
```

Comment:
If you do these as fixtures (you can define test-specific fixtures directly in the test file just like you do in `conftest.py`), your code won't have to drift as far to the right.

Reply:
Or, probably better, a single fixture, since they don't really make sense by themselves. Basically just do the same series of `with` statements but then run `yield` in the body, and you have a yield fixture that yields `None` that you can then reference in the test as `mock_git_lfs: None` or whatever as an argument.